### PR TITLE
cleaned up code & edited README

### PR DESCRIPTION
**Pull request by AnaLanda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Online Music Instrument Store
 
-An online store with basic functionality for users and admins written in Java. 
+A prototype of an online store with basic functionality for users and admins written in Java. 
 Store clients can perform the following actions:
```
Comment (andromash) [inline README.md:14]: I think it will be better to show functionality after the project structure

File: README.md [hunk 2]
```diff
@@ -13,6 +13,11 @@ Admins in their turn can:
  - delete users from the store's database;
  - view all orders placed at the store; 
  - edit the store's stock of instruments.
+ 
+## Project Structure
+The project has an N-tier structure and consists of the database layer, the DAO layer for interaction with the database, the service layer which contains the business logic, and the presentation layer.
+Servlets are used to receive and respond to client requests; filters control access to the store's functionality; and the presentation layer is implemented with JSP and Bootstrap. The DAO layer includes two implementations - one for the JDBC, which is used for work with a database; and one where data is stored locally in Lists. 
+This project also includes custom-made annotations and an injector, which utilizes Reflection API.
```
Comment (andromash) [inline README.md]: Was not able to write next, but I think MySQL Connector is not enough, better write MySQL RDMS and JDBC

**Review by andromash — COMMENTED** [review]
Nice job!

File: README.md [hunk 3]
```diff
@@ -37,4 +44,12 @@ Replicate the database from the project by copying the script from init_db.sql i
...
\ No newline at end of file
+
+When you launch the website for the first time, click on "Inject data" to add the user and admin data to the DB so that the store works properly.
+
+To log in as a User on the website without registration, you can log in as _imp_ with the password _password1234_. 
+
+To log in as an Admin on the website, you should log in as _admin_ with the password _1234_. 
+
+## Author
+([Anastasiia Melnyk(https://github.com/AnaLanda))
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
[Anastasiia Melnyk](https://github.com/AnaLanda)
```

File: README.md [hunk 4]
```diff
@@ -37,4 +44,12 @@ Replicate the database from the project by copying the script from init_db.sql i
...
 
 Your MySQL server must be up and running when you launch the project.
- 
\ No newline at end of file
+
+When you launch the website for the first time, click on "Inject data" to add the user and admin data to the DB so that the store works properly.
+
+To log in as a User on the website without registration, you can log in as _imp_ with the password _password1234_. 
+
+To log in as an Admin on the website, you should log in as _admin_ with the password _1234_.
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
To log in as an Admin on the website, you should log in as `admin` with the password `1234`. 
```

File: README.md [hunk 5]
```diff
@@ -37,4 +44,12 @@ Replicate the database from the project by copying the script from init_db.sql i
 Insert your own MySQL username and login in dbProperties in the ConnectionUtil class. 
 
 Your MySQL server must be up and running when you launch the project.
- 
\ No newline at end of file
+
+When you launch the website for the first time, click on "Inject data" to add the user and admin data to the DB so that the store works properly.
+
+To log in as a User on the website without registration, you can log in as _imp_ with the password _password1234_.
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
To log in as a User on the website without registration, you can log in as `imp` with the password `password1234`. 
```

File: README.md [hunk 6]
```diff
@@ -1,19 +1,26 @@
 # Online Music Instrument Store
 
-An online store with basic functionality for users and admins written in Java. 
+A prototype of an online store with basic functionality for users and admins written in Java. 
+
+## Project Structure
+The project has an N-tier structure and consists of the database layer, the DAO layer for interaction with the database, the service layer which contains the business logic, and the presentation layer.
+Servlets are used to receive and respond to client requests; filters control access to the store's functionality; and the presentation layer is implemented with JSP and Bootstrap. The DAO layer includes two implementations - one for the JDBC, which is used for interaction with the database; and one where data is stored locally in Lists. 
+This project also includes custom-made annotations and an injector, which utilizes Reflection API.
```
Comment (KseniiaMakarova) [inline README.md]: this text is now presented as a single block, let's spread it out a little to improve readability

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**